### ISSUE TITLE
mysqld_exporter doesn't start up

### ISSUE THREAD

**Issue by limed** [nubisproject/nubis-db-admin#48]
Found out that mysqld_exporter is not starting up journalctl tells me this

```
Dec 06 22:29:18 ip-172-16-148-22.us-west-2.compute.internal mysqld_exporter[18441]: time="2017-12-06T22:29:18Z" level=info msg="Starting mysqld_exporter (version=0.9.0, branch=master, revision=8400af20ccdbf6b5e0faa2c925c56c48cd78d70b)" source="mysqld_exporter.go:432"
Dec 06 22:29:18 ip-172-16-148-22.us-west-2.compute.internal mysqld_exporter[18441]: time="2017-12-06T22:29:18Z" level=info msg="Build context (go=go1.6.3, user=root@2c131c66ca20, date=20160926-18:28:09)" source="mysqld_exporter.go:433"
Dec 06 22:29:18 ip-172-16-148-22.us-west-2.compute.internal mysqld_exporter[18441]: time="2017-12-06T22:29:18Z" level=fatal msg="failed reading ini file: open .my.cnf: no such file or directory" source="mysqld_exporter.go:439"
```

Based on the config it says `-config.my-cnf` defaults to `/root/.my.cnf` which is obviously a lie

**Comment by gozer** [comment]
I suspect it's because under systemd we are missing something like $HOME or such, bummer

**Comment by gozer** [comment]
Fixed by #49
